<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SubForm3Base</class>
 <widget class="QWidget" name="SubForm3Base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configuracion Subformularios</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget2" name="mui_list" />
   </item>
   <item>
    <widget class="QFrame" name="mui_configurador" >
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="midLineWidth" >
      <number>2</number>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="mui_detalles" >
          <property name="minimumSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Imprimir pedido</string>
          </property>
          <property name="whatsThis" >
           <string>Imprimir pedido</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="mui_selectorsubform3" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">asdf&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="accessibleName" >
         <string>Query</string>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <widget class="QWidget" name="tab" >
         <attribute name="title" >
          <string>Columnas</string>
         </attribute>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="mui_confcol" >
            <property name="text" >
             <string>Aplicar Cambios</string>
            </property>
            <property name="icon" >
             <iconset resource="../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_reload.png</iconset>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="mui_listcolumnas" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2" >
         <attribute name="title" >
          <string>Query</string>
         </attribute>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="mui_confquery" >
            <property name="text" >
             <string>Aplicar Cambios</string>
            </property>
            <property name="icon" >
             <iconset resource="../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_reload.png</iconset>
            </property>
            <property name="toolButtonStyle" >
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="mui_query" >
            <property name="font" >
             <font>
              <family>Sans Serif</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="cursor" >
             <cursor>4</cursor>
            </property>
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="acceptDrops" >
             <bool>false</bool>
            </property>
            <property name="autoFormatting" >
             <set>QTextEdit::AutoAll</set>
            </property>
            <property name="tabChangesFocus" >
             <bool>true</bool>
            </property>
            <property name="documentTitle" >
             <string comment="Consulta SQL" >Query</string>
            </property>
            <property name="readOnly" >
             <bool>false</bool>
            </property>
            <property name="acceptRichText" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QTableWidget2</class>
   <extends>QTableWidget</extends>
   <header>qtable2.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bulmages-recursos-graficos.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_detalles</sender>
   <signal>clicked()</signal>
   <receiver>mui_configurador</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>642</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel" >
     <x>487</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
